spec/controllers/course/assessment/submission/submissions_controller_spec.rb

```diff
@@ -76,10 +75,9 @@
           subject
         end
 
-        it { is_expected.to redirect_to(course_assessments_path(course)) }
-        it 'sets the proper flash message' do
-          expect(flash[:danger]).to eq(I18n.t('course.assessment.submission.submissions.create.'\
-                                              'failure', error: ''))
+        it do
+          is_expected.to have_http_status(:bad_request)
+          expect(JSON.parse(response.body)['error']).not_to be_nil
```

Comment:
use `.to eq`

Reply:
do we need to check the error message as well? I thought as long as the error message is there, we are good?

Reply:
You can try to byebug and check whats the error message

Reply:
done